### PR TITLE
Use Heroku Local (Forego) instead of Foreman

### PR DESCRIPTION
**Pull request by croaky**
Heroku Local has replaced Foreman in the Heroku Toolbelt.

https://devcenter.heroku.com/changelog-items/692

The command `heroku local` has replaced `foreman`.
Heroku Local makes use of [Forego] to accomplish its tasks
and it is faster and has better cross-platform support.

[Forego]: https://github.com/ddollar/forego

### CODE REVIEW

File: templates/sample.env [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# http://ddollar.github.com/foreman/
+# http://ddollar.github.com/forego/
```
Comment (mehlah) [inline templates/sample.env]: I guess this line can be :scissors:

File: lib/suspenders/app_builder.rb [hunk 2]
```diff
@@ -304,7 +304,7 @@ def configure_puma
       copy_file "puma.rb", "config/puma.rb"
     end
 
-    def setup_foreman
+    def setup_forego
```
Comment (zamith) [inline lib/suspenders/app_builder.rb]: Should this be renamed to `set_up_forego`? /cc @mcmire

Comment (mcmire) [inline lib/suspenders/app_builder.rb]: Yes, please :)

**Comment by tute** [comment]
After the grammar fix, LGTM!

File: templates/sample.env (hunk 1 above)
Comment (croaky) [inline templates/sample.env]: Good catch, I just changed to link to the correct Forego GitHub page.

File: lib/suspenders/app_builder.rb (hunk 2 above)
Comment (croaky) [inline lib/suspenders/app_builder.rb]: :paperclip: :100: :pencil2: :memo: I like it.

**Pull request merged**